cmd/snap/cmd_list_test.go

```diff
@@ -28,6 +28,26 @@ import (
 	snap "github.com/snapcore/snapd/cmd/snap"
 )
 
+func (s *SnapSuite) TestListHelp(c *check.C) {
```

Comment:
The test would highlight the flag is not documented. I agree about some kind of churn but this is also a way to see how people experience the command in some way